app/dashboards/achievement_dashboard.rb

```diff
@@ -12,9 +12,10 @@ class AchievementDashboard < Administrate::BaseDashboard
     user: Field::BelongsTo,
     supporting_evidence_attachment: Field::HasOne,
     supporting_evidence_blob: Field::HasOne,
+    self_verification_info: Field::String,
     current_state: Field::String.with_options(searchable: false),
     id: Field::String,
-    created_at: Field::DateTime,
+    created_at: Field::DateTime.with_options(format: "%d/%m/%Y %H:%M"),
```

Comment:
Why do we specify a format only here?

If we're going to use a particular format, could we use it consistently throughout the admin panel.

Presumably you could define a constant `FORMATTED_DATE_TIME = Field::DateTime.with_options(format: "%d/%m/%Y %H:%M")` in the base dashboard

Reply:
Good point, I did it on this one to make the table more readable, but we should be consistent, I have created a base dashboard and put in the suggested constant and updated the rest of the dashboards